researcher_ids/RI_Claims_V1.md

```diff
@@ -279,41 +393,162 @@ interfaces, such as:
+{
+  "iss": "<issuer-URL>",
+  "sub": "<subject-identifier>",
+  ["scope": "openid ...",]
+  "jti": "<token-identifier>",
+  "iat": <seconds-since-epoch>,
+  "exp": <seconds-since-epoch>,
+  "ga4gh_visa_v1": {
+    "type": "<passport-visa-type>",
+    "asserted": <seconds-since-epoch>,
+    "value": "<value-string>",
+    "source": "<source-URL>",
+    ["condition": {...},]
+    ["by": "<by-identifier>"]
+  }
+}.<signature>
+```
+
+Where fields within the `ga4gh_visa_v1` object are as described in the
+[Passport Visa Fields](#passport-visa-fields) section of this
+specification except for "exp" which is a direct JWT claim and not
+stored within the `ga4gh_visa_v1` object.
+
+One of `scope` or `jku` MUST be present as described in
+[Conformance for Embedded Token Signatories](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#conformance-for-embedded-token-signatories)
+within the [AAI specification](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md).
+
+#### "**exp**"
+
+-   REQUIRED. Generally, it is seconds since unix epoch of when the
+    [Passport Assertion Source](#passport-assertion-source) requires such a
+    claim to be no longer valid. A Passport Assertion Source MAY choose
+    to make a claim very long lived. However, a [Passport Visa
+    Signatory](#passport-visa-signatory) MAY choose an earlier timestamp
+    in order to limit the claim’s duration within downstream Passport
+    Clearinghouses.
+
+-   Access is NOT necessarily removed by the `exp` timestamp. Instead,
+    this timestamp may be viewed as a cut-off after which no new access
+    will be granted and action to remove any existing access may
+    commence anytime shortly after this cut-off period.
+
+-   Its use by a Passport Clearinghouse is described in the
+    [Passport Visa Expiry](#passport-visa-expiry) section and
+    [Token Revocation](#token-revocation) section.
+
+## Passport Visa Fields
+
+Although standard fields within a [Passport Visa Object](#passport-visa-object)
+are defined in this section, other fields MAY exist within the object
+and should be ignored by any Passport Clearinghouse that is not familiar
+with the use of such fields. Field names are reserved for definition by
+the GA4GH DURI committee.
+
+#### "**type**"
+
+-   REQUIRED. A [Passport Visa Type](#passport-visa-type) that is
+    either a [GA4GH Standard Passport Visa
+    Definition](#ga4gh-standard-passport-visa-definition) name, or a
+    [Custom Passport Visa Type](#custom-passport-visa-types) name.
+
+#### "**asserted**"
+
+-   REQUIRED. Seconds since unix epoch that represents when the [Passport
+    Assertion Source](#passport-assertion-source) made the claim.
+
+-   Its use by a Passport Clearinghouse is described in the
+    [Passport Visa Expiry](#passport-visa-expiry) section.
+
+-   If a [Passport Assertion Repository](#passport-assertion-repository)
+    does not include enough information to construct an `iat` timestamp, a
```

Comment:
Can you cross-check the use of "asserted" and "iat". Is the difference between the two described somewhere?

Reply:
Fixed text in the spec. There are more details in the landing page.